src/llm/model.py

```diff
@@ -0,0 +1,283 @@
+from typing import List
+import torch
+import math
+from torch import Tensor
+from torch import nn
+from dataclasses import dataclass
+from jaxtyping import Float32
+
+
+@dataclass
+class TransformerConfig:
+    model_dim: int
+    expansion_dim: int
+    num_heads: int
+    num_layers: int
+
+
+class FeedForward(nn.Module):
+    def __init__(
+        self,
+        model_dim: int,
+        expansion_dim: int,
+    ):
+        super().__init__()
+        self.layer1 = nn.Linear(model_dim, expansion_dim, bias=True)
+        self.layer2 = nn.Linear(expansion_dim, model_dim, bias=True)
+
```

Comment:
`self.activation = nn.ReLU()`

from the paper: 

"This consists of two linear transformations with a ReLU activation in between"

Reply:
Great catch! Completely missed this.